scripts/devnet-consts.json

```diff
@@ -241,6 +241,17 @@
                 }
             }
         },
+        "21": {
+            "contracts": {
+                "tokenBridgeEmitterAddress": "0000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
NOTE: This will need to be the `EmitterCap` id (found in Token Bridge's `State`)